tests/rule_based_profiler/bobby_user_workflow_verbose_profiler_config.yml

```
@@ -9,11 +9,13 @@ variables:
     data_asset_name: my_reports
     data_connector_query:
       index: ":-1"
+  sampling_method: oneshot  # Choices: bootstrap (default) or oneshot (inaccurate -- should be used only for testing).
+  nan_raises_exception: False  # Choices: False (default) or True (if False, then NaN is converted to 0.0).
```

Just an optional stylistic suggestion, feels more explicit for the parameter to describe the action. Something like:
```suggestion
  fill_nan_with_zero: True  # Choices: True (default) or False (if True, then NaN is converted to 0.0).
```

---

@anthonyburdi Thank you for this suggestion -- I have been searching for a good parameter name, and you found it!  (This change will be incorporated as part of the next commit, and I will notify you.  Thanks!)